backend/ctgov/api/views.py

```diff
@@ -676,3 +743,514 @@ def filter_phase(request):
+        str(request.data.get("study_funder_type"))
+        if request.data.get("study_funder_type")
+        else ""
+    )
+    study_document_type = (
+        str(request.data.get("study_document_type"))
+        if request.data.get("study_document_type")
+        else ""
+    )
+
+    study_results_submitted = request.data.get("study_results_submitted")
+    query_type_qc_not_done = ""
+    query_type_qc_done = ""
+
+    if study_results_submitted:
+        if str(study_results_submitted).lower() == "not submitted":
+            query_type_qc_not_done = "null"
+        elif str(study_results_submitted).lower() == "submitted":
+            query_type_qc_not_done = "not null"
+        else:
+            query_type_qc_done = "not null"
+
+    study_title_acronym = (
+        str(request.data.get("study_title_acronym"))
+        if request.data.get("study_title_acronym")
+        else ""
+    )
+    study_outcome_measure = (
+        str(request.data.get("study_outcome_measure"))
+        if request.data.get("study_outcome_measure")
+        else ""
+    )
+
+    eligibility_age = request.data.get("eligibility_age")
+    if valid_number(eligibility_age):
+        eligibility_age_lower_limit = convert_to_number(eligibility_age)
+        eligibility_age_upper_limit = convert_to_number(eligibility_age)
+    else:
+        eligibility_age_lower_limit = 0
+        eligibility_age_upper_limit = 200
+
+    eligibility_age_group = request.data.get("eligibility_age_group")
+    if eligibility_age_group:
+        age_group_list = str(eligibility_age_group).split(",")
+        for age_group in age_group_list:
+            if age_group.lower().strip(" ") == "child":
+                eligibility_age_group_lower_limit_1 = 0
+                eligibility_age_group_lower_limit_2 = 0
+                eligibility_age_group_upper_limit_1 = 17
+                eligibility_age_group_upper_limit_2 = 17
+            if age_group.lower().strip(" ") == "adult":
+                eligibility_age_group_lower_limit_1 = 0
+                eligibility_age_group_lower_limit_2 = 18
+                eligibility_age_group_upper_limit_1 = 0
+                eligibility_age_group_upper_limit_2 = 65
+            if age_group.lower().strip(" ") == "older adult":
+                eligibility_age_group_lower_limit_1 = 0
+                eligibility_age_group_lower_limit_2 = 65
+                eligibility_age_group_upper_limit_1 = 0
+                eligibility_age_group_upper_limit_2 = 200
+    else:
+        eligibility_age_group_lower_limit_1 = 0
+        eligibility_age_group_lower_limit_2 = 0
+        eligibility_age_group_upper_limit_1 = 200
+        eligibility_age_group_upper_limit_2 = 200
+
+    study_roa = (
+        request.data.get("study_roa") if request.data.get("study_roa") else ""
+    )
+    phase = request.data.get("phase") if request.data.get("phase") else ""
+
+    # Construct filter json object
+    filters = (
+        "{ "
+        + '"from" :'
```

Comment:
Might make more sense here to build a dictionary then use `json.dumps(dictionary)`. This would eliminate a lot of the possibility for human error when manually constructing this JSON.